testing/features/step_definitions/navigation_steps.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+Given("the navigation is on the page") do
+  @component = Navigation::Default.new.tap(&:load)
+end
+
+Given("the navigation is on the page with header links") do 
+  @component = Navigation::WithHeader.new.tap(&:load)
+end
+
+Given("the navigation does not fit on the screen") do 
```

Comment:
We have a tag which will do this for you. I believe it is something like `@requires_small_screen` or something similar.